.github/workflows/maven.yml

```diff
@@ -5,22 +5,74 @@ name: Java CI with Maven
 
 on: [push, pull_request]
 
+env:
```

Comment:
did you start with the most recent develop branch?  There seems like too many changes.   Not to mention all the formatting changes that makes finding the real differences hard.

Reply:
for gui the base branch is https://github.com/informatici/openhospital-gui/tree/OP-1200-Wards-Inventory, that is the one to be updated now, let me do it